src/entity/active_enum.rs

```diff
@@ -275,6 +275,64 @@ mod tests {
         assert_eq!(format!("{}", DeriveCategory::Small), "'S'");
     }
 
+    #[test]
+    fn display_mode() {
+        // Completely default -> All print values
+        #[derive(PartialEq, Eq, EnumIter, DeriveActiveEnum)]
+        #[sea_orm(rs_type = "i32", db_type = "Integer")]
+        enum DefaultDisplay {
+            One = 1,
+            Two = 2,
+            Three = 3,
+        }
+
+        assert_eq!(format!("{}", DefaultDisplay::One), "1");
+        assert_eq!(format!("{}", DefaultDisplay::Two), "2");
+        assert_eq!(format!("{}", DefaultDisplay::Three), "3");
+
+        // All `label` -> All print label
+        #[derive(PartialEq, Eq, EnumIter, DeriveActiveEnum)]
+        #[sea_orm(rs_type = "i32", db_type = "Integer", display = "label")]
```

Comment:
"label" can be the default. May be we can make `display` an alias of `display = "label"`.

Reply:
as discussed, we want `display = label | value | "string literal"` but i cannot get it working, so instead went to `display = "string literal" | display_value | display_label`.